rust/template/src/ovsdb_api.rs

```diff
@@ -342,9 +350,13 @@ fn dump_output(
                     .map(|(v, w)| {
                         assert!(*w == 1 || *w == -1);
                         if (*w == 1) {
-                            record_into_insert_str(v.clone().into_record(), table_str)
+                            record_into_insert_str(v.clone().into_record(), table_str, &|t| {
+                                prog.get_table_original_name(t)
+                            })
                         } else {
-                            record_into_delete_str(v.clone().into_record(), table_str)
+                            record_into_delete_str(v.clone().into_record(), table_str, &|t| {
+                                prog.get_table_original_name(t)
+                            })
                         }
```

Comment:
These branches are incredibly similar
```suggestion
                        let recorder = if (*w == 1) {
                            record_into_insert_str
                        } else {
                            record_into_delete_str
                        };
                        let get_table_name = |table| prog.get_table_original_name(t);
                        let record = v.clone().into_record();

                        recorder(record, table_str, get_table_name);
```